README.md

```diff
@@ -297,6 +291,18 @@ If you find a bug in Puppet Lint or its results, please create an issue in the
 [repo issues tracker](https://github.com/puppetlabs/puppet-lint/issues/). Bonus
 points will be awarded if you also include a patch that fixes the issue.
 
+## Development
+
+Acceptance tests for this tool leverage [puppet_litmus](https://github.com/puppetlabs/puppet_litmus).
```

Comment:
I really wish it didn't. Litmus an aweful tool for this and I wish https://github.com/puppetlabs/puppet-lint/pull/89 got a proper response for maintainers.

Reply:
Hi @ekohl. I'll make a note of this and bring it to the team for discussion